documentation/docs/concepts/decentralized_identifiers/resolve.mdx

```diff
@@ -6,187 +6,148 @@ image: /img/Identity_icon.png
 keywords:
 - Resolve
 ---
-import resolve_did_rs from '!!raw-loader!../../../../examples_legacy/low-level-api/resolve_did.rs';
-import resolve_history_rs from '!!raw-loader!../../../../examples_legacy/low-level-api/resolve_history.rs';
-import resolve_did_js from '!!raw-loader!../../../../bindings/wasm/examples/src/resolve_did.js';
-import resolve_history_js from '!!raw-loader!../../../../bindings/wasm/examples/src/resolve_history.js';
-import CodeSnippet from '../../../src/components/CodeSnippetComponent'
+import resolve_did_rs from '!!raw-loader!../../../../examples/0_basic/2_resolve_did.rs';
+import resolve_did_js from '!!raw-loader!../../../../bindings/wasm/examples-stardust/src/ex2_resolve_did.ts';
+import CodeBlock from '@theme/CodeBlock';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
-import CodeBlock from '@theme/CodeBlock';
-
-
-DID resolution is the process of fetching a [DID Document](https://www.w3.org/TR/did-core/#dfn-did-documents) corresponding to a given [DID](https://www.w3.org/TR/did-core/#dfn-decentralized-identifiers).
-The [IOTA Identity Framework](https://github.com/iotaledger/identity.rs) supports resolving DID Documents that are stored on an IOTA Tangle (public or private). The main tool supplied
-by the IOTA Identity Framework to handle DID Document resolution in a type safe manner is the `Resolver`. A DID Resolver as defined in the [W3C Decentralized Identifiers specification](https://www.w3.org/TR/did-core/#dfn-did-resolvers)
-enforces the signature of the resolution function in a manner that is more centered around Web/API resolution rather than a strongly typed framework. This is the reason why the `Resolver` provided by the IOTA Identity Framework deviates somewhat from
-the W3C specification.
-
-
-## Resolving a DID from the main network
-The following example demonstrates how to resolve the DID: "did:iota:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV" from the `main` network.
-<Tabs>
-<TabItem value="Rust" label="Rust">
-
-```rust
-use identity_iota::client::Resolver;
-use identity_iota::iota_core::IotaDID;
-use identity_iota::client::ResolvedIotaDocument;
 
-let resolver: Resolver = Resolver::new().await?;
-let did: IotaDID = IotaDID::parse("did:iota:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV")?;
+DID resolution is the process of fetching and decoding a [DID Document](https://www.w3.org/TR/did-core/#dfn-did-documents) corresponding to a given [DID](https://www.w3.org/TR/did-core/#dfn-decentralized-identifiers). The [IOTA Identity framework](https://github.com/iotaledger/identity.rs) supports resolving DID Documents that are stored on the IOTA Tangle and Shimmer networks.
```

Comment:
Shall we use `Ledger` instead of `Tangle`?

Reply:
I think just `IOTA network` should be the preferred terminology, since the wiki defines "IOTA" and "Shimmer" as "Networks". I haven't seen `IOTA Ledger` used elsewhere.